lib/torba/verify.rb

```diff
@@ -1,7 +1,14 @@
 require "torba"
 
-if STDOUT.tty?
-  Torba.pretty_errors { Torba.verify }
-else
-  Torba.verify
+# We purposely skip verification during Rake execution, because Rake is used to
+# run the Torba:RakeTask during the "assets:precompile" stage of deployment. In
+# this case it is OK that the Torbafile hasn't been packed yet, because we are
+# about to do so.
+#
+unless defined?(Rake::Task)
```

Comment:
Unfortunately, this doesn't work if I run `bin/rake assets:precompile` or `bundle exec rake assets:precompile`.
Could we also/instead check if `$0` ends with "rake" to cover migration/compilation tasks?
Keep in mind that in Rails 5 one [can use](https://github.com/rails/rails/commit/3e65c3d3886336e9145438cdeacaf4ebec6a48b8) `bin/rails assets:precompile`